querybook/server/datasources/metastore.py

```diff
@@ -665,3 +665,31 @@ def get_schemas(metastore_id, limit=5, offset=0, sort_key="name", sort_order="de
     schemas = logic.get_all_schemas(metastore_id, offset, limit, sort_key, sort_order)
 
     return {"results": schemas, "done": len(schemas) < limit}
+
+
+@register("/metastore/<int:metastore_id>/sync/", methods=["PUT"])
```

Comment:
yep, can use the params instead of request body json.

but will need metastore_id anyway,  `/metastore/<int:metastore_id>/<str:schema_name>/<str:table_name>/sync/`